python/kvikio/kvikio/benchmarks/utils.py

```diff
@@ -41,6 +42,10 @@ def pprint_sys_info() -> None:
         gpu_name = f"{pynvml.nvmlDeviceGetName(dev)} (dev #0)"
         mem_total = format_bytes(pynvml.nvmlDeviceGetMemoryInfo(dev).total)
         bar1_total = format_bytes(pynvml.nvmlDeviceGetBAR1MemoryInfo(dev).bar1Total)
+    if version == (0, 0):
+        libcufile_version = "unknown"
```

Comment:
Maybe we can add this info here or perhaps just in a code comment.
```suggestion
        libcufile_version = "unknown (earlier than cuFile 1.8)"
```